ably/http/http.py

```diff
@@ -132,7 +135,7 @@ def make_request(self, method, path, headers=None, body=None,
                 self.options.use_binary_protocol)
 
         if not skip_auth:
-            if self.auth.auth_method == Auth.Method.BASIC and self.preferred_scheme.lower() == 'http':
+            if self.auth.auth_mechanism == Auth.Method.BASIC and self.preferred_scheme.lower() == 'http':
```

Comment:
But @aericson, `authMethod` is an attribute of `ClientOptions`.  I don't see the relevance to the auth object itself?